src/Tizen.NUI/src/public/XamlBinding/Transition.cs

```diff
@@ -10,6 +11,18 @@
 
 namespace Tizen.NUI
 {
+    /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class StoryBoard
```

Comment:
I think this class is not related with this patch.

Reply:
To fix the CA1819, I remove the property TranstionNames,
and because the transition has been refactor, so I use a new class to restore the transtion.

The name "StoryBoard" is refer to WPF.